apps/emqx_ds_shared_sub/src/emqx_ds_shared_sub_proto.erl

```diff
@@ -57,6 +51,20 @@
     agent_metadata/0
 ]).
 
+-define(log_agent_msg(ToLeader, Msg),
+    ?tp(debug, shared_sub_proto_msg, #{
```

Comment:
Logs that were at `warning` level now are `debug`.  Is it intentional?

Reply:
Yes